src/metrics.rs

```diff
@@ -49,7 +48,7 @@ pub enum ProcessTableSortBy {
     User = 1,
     Priority = 2,
     Nice = 3,
-    CPU = 4,
+    Cpu = 4,
```

Comment:
Hah, no strong opinion here, but was originally all capital letters since it is technically an abbreviation.

Reply:
Yes, it was just what clippy suggested, I was wondering if adding an `allow(clippy::...)` would be worth it.